arroyo-worker/src/connectors/webhook.rs

```diff
@@ -0,0 +1,187 @@
+use std::{
+    marker::PhantomData,
+    sync::Arc,
+    time::{Duration, SystemTime},
+};
+
+use arroyo_macro::process_fn;
+use arroyo_rpc::ControlResp;
+use arroyo_rpc::{grpc::TableDescriptor, OperatorConfig};
+use arroyo_state::tables::GlobalKeyedState;
+use arroyo_types::{string_to_map, CheckpointBarrier, Key, Record};
+use bincode::{Decode, Encode};
+
+use serde::{Deserialize, Serialize};
+use tokio::sync::{Mutex, Semaphore};
+
+use tracing::warn;
+use typify::import_types;
+
+use crate::{
+    engine::{Context, StreamNode},
+    formats::DataSerializer,
+    SchemaData,
+};
+
+import_types!(schema = "../connector-schemas/webhook/table.json");
+
+const MAX_INFLIGHT: u32 = 50;
+
+#[derive(Clone, Debug, Encode, Decode, PartialEq, PartialOrd, Default)]
+pub struct WebhookSinkState {}
```

Comment:
What's the point of an empty struct stored on the key `()`? It looks like this could be removed without affecting behavior. Leftover from a previous approach?

Reply:
Yeah, it can be removed